README.md

```diff
@@ -1,2 +1,18 @@
-# Capstone-Project-1
- First capstone project at Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project 1
+
+> This is my first capstone project while studying at Microverse. It's a simple website for a festival. You can check it out by clicking this link: https://thi-ponce.github.io/Capstone-Project-1/
+> You can also see me presenting this project in a short time by clicking here: https://www.loom.com/share/37c8786a067349ee940345752908de6a
+
+## Built With
+
+- HTML, CSS, Js
+- VsCode
+## Authors
+
+👤 **Thiago Ponce**
+
+- GitHub: [@Thi-Ponce](https://github.com/Thi-Ponce)
+- Twitter: [@Thi_ponce](https://twitter.com/Thi_ponce)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/thiago-ponce)
```

Comment:
- [ ] As the previous reviewer said don't forget to add Add descriptive README files to your project. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it with your project information. Kindly add some important sections in your Readme file like: ` Contributing`, `Acknowledgments`, ` License`, `Live Demo (if available)`, `Getting Started` referring yourself to [this template](https://github.com/microverseinc/readme-template).